source/includes/api/_before_you_get_started.md

```diff
@@ -56,6 +56,130 @@ Below you can find a list of RW API resources that use the `application` field:
+* `MANAGER` accounts can perform all of the `USER` actions, complemented with editing or deleting resources owned by them;
+* `ADMIN` accounts can do all of the above even for resources created by other users.
+
+Role-based access control is usually conjugated with the list of applications associated with the user: typically, in order to manipulate a given resource, that resource and the user account must have at least one overlapping application value. Read more about the application field and which services use it in the [Applications concept documentation](/index-rw.html#applications).
+
+Keep in mind that it’s up to each individual RW API service (dataset, widget, layer, etc) to define how they restrict or allow actions based on these or other factors, so the examples above may not be true for all cases. Refer to the documentation of each resource and endpoint for more details on restrictions they may have regarding user accounts and their properties.
+
+### How to change the role of an user
+
+Changing role of users is restricted to `ADMIN` users, so if you intend to upgrade your user role to a `MANAGER` or `ADMIN` role, please get in touch with one of the `ADMIN` users and request the change. If you are already an `ADMIN` user and you intend to change the role of another user, you can do so using the [`PATCH /users/:id` endpoint](/index-rw.html#update-another-user-39-s-account-details).
+
+### Which services comply with these guidelines
+
+The following endpoints adhere to the Sorting conventions defined above:
+
+* [Dashboards](/index-rw.html#dashboard)
+* [Datasets](/index-rw.html#dataset6)
+* [Layers](/index-rw.html#layer8)
+* [Metadata](/index-rw.html#metadata14)
+* [Widgets](/index-rw.html#widget9)
+
+## Sorting
+
+> Example request sorting by a single condition:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?sort=name
+```
+
+> Example request sorting by multiple conditions:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?sort=name,description
+```
+
+> Example request sorting by multiple conditions, descending and ascending:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?sort=-name,+description
+```
+
+As a rule of thumb, you can sort RW API resources using the `sort` query parameter. Usually, sorting can be performed using any field from the resource schema, so be sure to check each resource's model reference to find which fields can be used for sorting. In some cases, you also have the possibility of sorting by fields that are not present in the resource model (e.g., when fetching datasets, you can sort by `user.name` and `user.role` to sort datasets by the name or role of the owner of the dataset) - check each resource's documentation to find out which additional sorting criteria you have available. Sorting by nested fields is not supported at the moment.
```

Comment:
That nested fields note is confusing right after talking about `user.name`. Maybe just say: "Sorting by nested fields is not generally supported, but may be implemented for particular resources."

Reply:
I've accepted your suggestion, and I've reworded the paragraph a bit to make it less confusing :)